reference/forms/types/options/choice_attr.rst.inc

```diff
@@ -4,13 +4,12 @@ choice_attr
 .. versionadded:: 2.7
     The ``choice_attr`` option was introduced in Symfony 2.7.
 
-**type**: ``array``, ``callable`` or ``string`` **default**: ``array()``
+**type**: ``array``, ``callable``, ``string`` or :class:``Symfony\Component\PropertyAccess\PropertyPath`` **default**: ``array()``
 
-Use this to add additional HTML attributes to each choice. This can be an array
-of attributes (if they are the same for each choice), a callable or a property path
```

Comment:
I guess this might be one reason why someone would end like symfony/symfony#17019